library/altera/common/up_rst_constr.sdc

```diff
@@ -1,6 +1,6 @@
 
-set_false_path  -from [get_registers *up_*rst_async*] -to [get_registers *ad_rst:i_core_rst_reg|rst_sync_d]
-set_false_path  -from [get_registers *up_*rst_async*] -to [get_registers *ad_rst:i_core_rst_reg|rstn]
-
-set_false_path  -from [get_registers *up_core_preset] -to [get_registers *ad_rst:i_core_rst_reg|rst_async_d*]
+set_false_path  -to [get_registers *ad_rst:*|rst_async_d*]
+set_false_path  -to [get_registers *ad_rst:*|rst_sync]
+set_false_path  -to [get_registers *ad_rst:*|rst_sync_d]
+set_false_path  -to [get_registers *ad_rst:*|rst]
```

Comment:
The last two should not have a false path. They are your synchronization registers to prevent meta stability at the output. In fact they should be placed as close to each other as possible.

Strictly speaking the only false paths are to the CLR pin of rst_async_d1, rst_async_d2 and rst_sync.

Reply:
Yes, you're right. Thanks.